flags.go

```diff
@@ -255,6 +255,13 @@ func newApp() (app *cli.App) {
 				Usage: "The format of the log file: 'text' or 'json'.",
 			},
 
+			cli.BoolTFlag{
+				Name: "log-fuse-errors",
```

Comment:
should we rename this to debug_fuse_errors to be consistent with debug_fuse name?

Reply:
Done